pkg/config/config.go

```diff
@@ -266,7 +266,6 @@ func (c *Config) GetStepConfig(flagValues map[string]interface{}, paramJSON stri
 			return StepConfig{}, err
 		}
 		if vaultClient != nil {
-			defer vaultClient.MustRevokeToken()
```

Comment:
I'd then add a comment here that states that the token will be revoked at the end of the step.
If it's very important to revoke the token (would be good to check), then I'd also comment that above the `GetVaultClientFromConfig()` function.